package.json

```diff
@@ -97,11 +89,11 @@
     "react-router-prop-types": "^1.0.4"
   },
   "peerDependencies": {
-    "@folio/stripes": "^8.0.0",
+    "@folio/stripes": "^9.0.0",
     "final-form": "^4.18.2",
     "final-form-arrays": "^3.0.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^18.2.0",
+    "react-dom": "^118.2.0",
```

Comment:
118

Reply:
Yep, all three of us missed it here 😬 . @vashjs fixed it for us in #105 when he published the release.